server/src/main/java/org/apache/iotdb/db/sync/sender/manage/ISyncFileManager.java

```diff
@@ -55,13 +55,14 @@
    */
   void getValidFiles(String dataDir) throws IOException;
 
-  Map<String, Map<Long, Set<File>>> getCurrentSealedLocalFilesMap();
 
-  Map<String, Map<Long, Set<File>>> getLastLocalFilesMap();
+  Map<String, Map<Long, Map<Long, Set<File>>>> getCurrentSealedLocalFilesMap();
```

Comment:
this also needs comments... @return

Reply:
virtual sg configuration in sg level is needed. Then the receiver could follow the configuration of the sender and each sg should be synced by one sender.